libmate-desktop/meson.build

```diff
@@ -0,0 +1,123 @@
+headers = [
+  'mate-bg-crossfade.h',
+  'mate-bg.h',
+  'mate-colorbutton.h',
+  'mate-colorsel.h',
+  'mate-colorseldialog.h',
+  'mate-dconf.h',
+  'mate-desktop-item.h',
+  'mate-desktop-thumbnail.h',
+  'mate-desktop-utils.h',
+  'mate-desktop.h',
+  'mate-gsettings.h',
+  'mate-languages.h',
+  'mate-rr-config.h',
+  'mate-rr-labeler.h',
+  'mate-rr.h',
+]
+
+introspection_src = files([
+  'display-name.c',
+  'edid-parse.c',
+  'mate-bg-crossfade.c',
+  'mate-colorbutton.c',
+  'mate-colorsel.c',
+  'mate-colorseldialog.c',
+  'mate-dconf.c',
+  'mate-desktop-thumbnail.c',
+  'mate-desktop-utils.c',
+  'mate-gsettings.c',
+  'mate-languages.c',
+  'mate-rr-config.c',
+  'mate-rr-labeler.c',
+  'mate-rr-output-info.c',
+  'mate-rr.c',
+  'mate-thumbnail-pixbuf-utils.c',
+  ])
+
+libmate_desktop_src = files([
+  'mate-desktop-item.c',
+  ]) + introspection_src
+
+if get_option('pnp-ids-path') == 'internal'
+  pnp_ids_path = join_paths(get_option('prefix'), get_option('datadir'), 'libmate-desktop/pnp.ids')
+  install_data('pnp.ids', install_dir: join_paths(get_option('datadir'), 'libmate-desktop'))
+else
+  pnp_ids_path = get_option('pnp-ids-path')
+endif
+
+libmate_desktop_inc = include_directories('.')
+libmate_desktop_lib = both_libraries('mate-desktop-2',
+  libmate_desktop_src,
+  c_args: [
+    '-DG_LOG_DOMAIN="MateDesktop"',
+    '-DISO_CODES_PREFIX="@0@"'.format(iso_codes_prefix),
+    '-DMATELOCALEDIR="@0@"'.format(join_paths(get_option('prefix'), get_option('localedir'))),
+    '-DMATE_DESKTOP_USE_UNSTABLE_API=1',
```

Comment:
Removing that line `'-DMATE_DESKTOP_USE_UNSTABLE_API=1',` fixes the build warnings.
So the question is why you want to add this here?

Reply:
Honestly unsure why, its been a bit since I looked at this. Had to reinstall my system due to a
mistake repartitioning.